### PR TITLE
Multiple mobile platforms

### PR DESCRIPTION
**Pull request by meandavejustice**
fixes 3672

txp-multi-platform-detail
txp-multi-platform

### CODE REVIEW

**Comment by meandavejustice** [comment]
@flodolo Can I get your approval on this one?

**Review by flodolo — APPROVED** [review]
Yup, missed this one.

File: frontend/src/app/containers/ExperimentPage/ExperimentControls.js [hunk 1]
```diff
@@ -165,8 +166,21 @@ function createButtons({
 
   const mobileControls = () => {
     if (!isMobile(userAgent)) {
-      return <MobileTriggerButton {...{ doShowMobileAppDialog, color: "default" }} />;
+      if (platforms.includes("ios")) {
```
Comment (lmorchard) [inline frontend/src/app/containers/ExperimentPage/ExperimentControls.js]: Seems like there's a little bit of redundant code here covering the ios, android, ios & android cases. You might be able to get away with just something like this:
```javascript
const mobileControls = () => (
  <Fragment>
    {(platforms.includes("ios")) &&
      <MobileTriggerIOSButton {...{ doShowMobileAppDialog, color: "default" }} />}
    {(platforms.includes("android")) &&
      <MobileTriggerAndroidButton {...{ doShowMobileAppDialog, color: "default" }} />}
  </Fragment>
);
```
(Didn't actually try this, so hopefully it's not a syntax error! 🤷‍♂️ )

File: frontend/src/app/components/MobileDialog/index.js [hunk 2]
```diff
@@ -72,14 +73,14 @@ export default class MobileDialog extends Component<MobileDialogProps, MobileDia
...
-    const headerMessage = ios_url
+    const headerMessage = isIOS
       ? <Localized id="mobileDialogMessageIOS" $title={title} b={<b></b>}>
         <p>Download <b>{title}</b> from the iOS App Store.</p>
       </Localized>
       : <Localized id="mobileDialogMessageAndroid" $title={title} b={<b></b>}>
         <p>Download <b>{title}</b> from the Google Play Store.</p>
       </Localized>;
-    const headerImg = ios_url ? (<a href={ios_url} onClick={handleAppLinkClick} target="_blank" rel="noopener noreferrer"><img className="mobile-header-img" src={iconIos}/></a>) : (<a href={android_url} onClick={handleAppLinkClick} target="_blank" rel="noopener noreferrer"><img className="mobile-header-img" src={iconGoogle}/></a>);
+    const headerImg = isIOS ? (<a href={ios_url} onClick={handleAppLinkClick} target="_blank" rel="noopener noreferrer"><img className="mobile-header-img" src={iconIos}/></a>) : (<a href={android_url} onClick={handleAppLinkClick} target="_blank" rel="noopener noreferrer"><img className="mobile-header-img" src={iconGoogle}/></a>);
```
Comment (lmorchard) [inline frontend/src/app/components/MobileDialog/index.js]: Just a nit: Might help to make this a little more readable if this got broken up over a couple lines. No biggie.

**Review by lmorchard — APPROVED** [review]
Just a couple of small nitpicks, but nothing I'd block merging over.

**Comment by meandavejustice** [comment]
Addressed comments, thanks @lmorchard

**Pull request merged**